lib/core/transpiler_session/attributed_type_map.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
-#include <clang/AST/ASTTypeTraits.h>
-#include <llvm/ADT/DenseSet.h>
+#include "../../../../../../../usr/lib/llvm-17/include/clang/AST/ASTTypeTraits.h"
```

Comment:
It looks not good, probably some debug footprints

Reply:
Nice find. I missed it probably during refactoring.

Reply:
Fixed.